frontend/src/views/camp/Admin.vue

```diff
@@ -11,15 +11,19 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
           <v-btn v-if="$vuetify.breakpoint.xsOnly" :to="{name: 'camp/collaborators', query: {isDetail: true}}">
             {{ $tc('views.camp.admin.collaborators') }}
           </v-btn>
+          <camp-periods :camp="camp" />
         </v-col>
         <v-col cols="12" lg="6">
-          <camp-periods :camp="camp" />
-
           <camp-categories :camp="camp" />
 
           <camp-material-lists :camp="camp" />
         </v-col>
       </v-row>
+      <v-row>
+        <v-col cols="12" lg="6">
+          <danger-zone :camp="camp" />
```

Comment:
I see you have drawn heavy inspiration from the GitHub danger zone. On GitHub however, the danger zone is far below the fold, while in eCamp, for now, the tempting red button is the closest of all buttons to the center of my screen. I'm not yet sure whether there will be a lot more settings coming to the Camp Admin page that will push the danger zone down. Have you considered to e.g. make the danger zone collapsed by default? Or at least not filled with red, similar to the outlined style of GitHub?

Reply:
I agree. I changed the buttons to outlined style and set the color to a subtle gray. Inside the dialog, when the name is filled in, the button will be the nice, tempting red ;)